lm_eval/models/anthropic_llms.py

```diff
@@ -43,7 +43,7 @@ def anthropic_completion(
     try:
         import anthropic
     except ModuleNotFoundError:
-        raise Exception(
+        raise ModuleNotFoundError(
```

Comment:
Just to reduce repetition, would this be better?
```
except ModuleNotFoundError as exception:
    raise exception(
        ...
    )
```

Reply:
This will fail due to `exception` is instance of error and not callable. See question https://stackoverflow.com/questions/9157210/how-do-i-raise-the-same-exception-with-a-custom-message-in-python 
I will check now if anything from these stackoverflow answers is helpful (provide more usable error handling) for such case.

Reply:
Yeah `exception` wasn't the right variable name for this context. Looks like `err` or something would work according to the answers.

Reply:
except ModuleNotFoundError as exception:
    raise type(exception)(
    ...
)
seems to work. I will update PR accordingly

Reply:
Updated, but now `exception` is repeated...